### PR TITLE
Make use of space in detail page

### PR DESCRIPTION
**Pull request by dannymichel**
Make use of space in detail page as seen here https://www.reddit.com/r/jellyfin/comments/f3sgpm/thoughts_on_105_web_changed/

Screen Shot 2020-02-14 at 10 52 02 AM

### CODE REVIEW

**Comment by anthonylavado** [comment]
Here are the lint issues specifically:
```
2020-02-14T16:06:39.8964853Z src/assets/css/librarybrowser.css
2020-02-14T16:06:39.8966260Z  1068:1   âœ–  Expected empty line before at-rule              at-rule-empty-line-before             
2020-02-14T16:06:39.8967083Z  1069:3   âœ–  Expected indentation of 4 spaces                indentation                           
2020-02-14T16:06:39.8968500Z  1070:7   âœ–  Expected indentation of 8 spaces                indentation                           
2020-02-14T16:06:39.8969040Z  1070:30  âœ–  Expected single space before "!"                declaration-bang-space-before         
2020-02-14T16:06:39.8969403Z  1071:7   âœ–  Expected indentation of 8 spaces                indentation                           
2020-02-14T16:06:39.8969810Z  1072:7   âœ–  Expected indentation of 8 spaces                indentation                           
2020-02-14T16:06:39.8970179Z  1073:7   âœ–  Expected indentation of 8 spaces                indentation                           
2020-02-14T16:06:39.8970585Z  1073:16  âœ–  Expected single space before "!"                declaration-bang-space-before         
2020-02-14T16:06:39.8970940Z  1074:3   âœ–  Expected indentation of 4 spaces                indentation                           
2020-02-14T16:06:39.8971334Z  1075:3   âœ–  Expected empty line before rule                 rule-empty-line-before                
2020-02-14T16:06:39.8971699Z  1075:3   âœ–  Expected indentation of 4 spaces                indentation                           
2020-02-14T16:06:39.8972094Z  1076:7   âœ–  Expected indentation of 8 spaces                indentation                           
2020-02-14T16:06:39.8972446Z  1077:3   âœ–  Expected indentation of 4 spaces                indentation                           
2020-02-14T16:06:39.8972843Z  1079:1   âœ–  Expected empty line before rule                 rule-empty-line-before                
2020-02-14T16:06:39.8973504Z  1079:14  âœ–  Expected double colon pseudo-element notation   selector-pseudo-element-colon-notation
```

**Comment by dkanada** [comment]
@grafixeyehero since you handled most of the design changes for this page I'll let you look over this pull request.

**Comment by nyanmisaka** [comment]
It seems that the logo conflicts with the button on the right.


conflict

**Comment by dannymichel** [comment]
I'm fixing the logo thing now

**Comment by dannymichel** [comment]
I'm not great at git stuff so i created another pr https://github.com/jellyfin/jellyfin-web/pull/803

File: src/assets/css/librarybrowser.css [hunk 1]
```diff
@@ -1062,6 +1052,31 @@ div:not(.sectionTitleContainer-cards) > .sectionTitle-cards {
...
+        float: right;
+    }
+}
+
+.itemBackdrop:after {
+    content: "";
+    width: 100%;
+    height: 100%;
+    background: #000000ad;
+    background: rgba(0, 0, 0, 0.65)
```
Comment (nyanmisaka) [inline src/assets/css/librarybrowser.css]: 1076:21  ✖  Missed semicolon   CssSyntaxError
```suggestion
    background: rgba(0, 0, 0, 0.65);
```

**Comment by nyanmisaka** [comment]
```
src/assets/css/librarybrowser.css
  574:16  ✖  Expected single space before "!"                                 declaration-bang-space-before         
  575:11  ✖  Expected single space after ":" with a single-line declaration   declaration-colon-space-after         
  575:18  ✖  Expected single space before "!"                                 declaration-bang-space-before         
  576:10  ✖  Expected single space after ":" with a single-line declaration   declaration-colon-space-after         
  576:17  ✖  Expected single space before "!"                                 declaration-bang-space-before         
 1062:5   ✖  Expected empty line before rule                                  rule-empty-line-before                
 1065:5   ✖  Expected empty line before rule                                  rule-empty-line-before                
 1071:14  ✖  Expected double colon pseudo-element notation                    selector-pseudo-element-colon-notation
```

**Comment by dannymichel** [comment]
should be fixed

**Comment by MrTimscampi** [comment]
Here's what I get when running your branch:

image

**Comment by dkanada** [comment]
@nyanmisaka we are very active on Matrix if you want to stop by our channels :smile: for both random discussions and development.

**Comment by dkanada** [comment]
I'm not sure moving the details bar to the middle is the right solution here, it makes the page look even more unorganized than it is now. I definitely agree the transparency change looks nice, what if you just applied that to the sticky and header together? @MrTimscampi wanted to add an option to hide the banner image I think, which would solve the other problem you had with the unused space.

**Comment by MrTimscampi** [comment]
I also have a few ideas to bounce. Let's try to build on this.

Here's the kind of thing I had in mind:

image

The height change for the top artwork block can be not so drastic, if needed.

On scroll, the poster would settle slightly below the bar and an IntersectionObserver would keep track of the sticky status and add a class that would slide the title/year/rating block further to the left.

**Comment by jaylex32** [comment]
This definitely looks better than how it is now.

**Comment by dannymichel** [comment]
For now i just  fixed the non transparent background

**Comment by dannymichel** [comment]
ok cool

**Comment by nvllsvm** [comment]
Relates to https://github.com/jellyfin/jellyfin-web/issues/820

**Comment by dkanada** [comment]
changes migrated to #823

**Pull request closed without merging**